synapse_net/training/domain_adaptation.py

```diff
@@ -18,6 +20,184 @@
 from ..inference.inference import get_model_path, compute_scale_from_voxel_size
 from ..inference.util import _Scaler
 
+class NewPseudoLabeler(self_training.DefaultPseudoLabeler):
```

Comment:
I suggest we give this a more descriptive name: `PseudoLabelerWithBackgroundMask`